src/TwigMiddleware.php

```diff
@@ -42,9 +36,18 @@ class TwigMiddleware implements MiddlewareInterface
     protected $basePath;
 
     /**
-     * @var string
+     * @param App    $app
```

Comment:
This param should be type-hinted to `ContainerInterface|null`.

Reply:
done